src/components/Table.js

```diff
@@ -497,6 +499,8 @@ export default class Table extends PureComponent {
 
   saveInnerRef = ref => {
     this.innerRef = ref;
+
+    this.props.innerRef(ref);
```

Comment:
Rather than setting `innerRef` as a function, do not provide any default value to it. Instead, check for it's existence here and then call it. No need for `noop` util if following this,